### PR TITLE
le model Users + controller sont crees

### PR DESCRIPTION
**Pull request by matthieu-laumet**
Can you check the model users and controller ?

### CODE REVIEW

File: db/migrate/20211116100457_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      t.string :last_name
+      t.string :email
+      t.string :phone_number
+      t.string :gender
+      t.string :adresse
+      t.string :driver_licence
+      t.string :username
+      t.string :password
+      t.string :type_of_car
+      t.bolean :driver
```
Comment (MelanieCouronne) [inline db/migrate/20211116100457_create_users.rb:14]: add a o to boolean

File: db/migrate/20211116100457_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
```
Comment (MelanieCouronne) [inline db/migrate/20211116100457_create_users.rb:1]: do you have launch the migration?

**Review by MelanieCouronne — CHANGES_REQUESTED** [review]

File: db/migrate/20211116100457_create_users.rb (hunk 1 above)
Comment (matthieu-laumet) [reply db/migrate/20211116100457_create_users.rb:14]: Ok it's done

File: db/migrate/20211116100457_create_users.rb (hunk 2 above)
Comment (matthieu-laumet) [reply db/migrate/20211116100457_create_users.rb:1]: Ok it's done

File: db/schema.rb [hunk 3]
```diff
@@ -10,9 +10,25 @@
...
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "users", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "email"
+    t.string "phone_number"
+    t.string "gender"
+    t.string "adresse"
```
Comment (chrismarchand) [inline db/schema.rb]: mettre address VS adresse svp :) <3 <3

**Review by chrismarchand — APPROVED** [review]

**Review by MelanieCouronne — APPROVED** [review]

**Pull request merged**